### PR TITLE
Add the --host option to override default mongodb ip binding via issue #17

### PR DESCRIPTION
**Pull request by chaiwa-berian**
This fixes issue #17

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -111,19 +119,19 @@ function* run() {
     if (result.set) {
       // There's already a replica set config, so don't initiate
```
Comment (chaiwa-berian) [inline index.js]:
```suggestion
       // There's already a replica set config, so don't initiate
```

File: index.js [hunk 2]
```diff
@@ -111,19 +119,19 @@ function* run() {
...
       yield rs.waitForPrimary();
```
Comment (chaiwa-berian) [inline index.js]:
```suggestion
       yield rs.waitForPrimary();
```

**Comment by Christilut** [comment]
This works :+1:

File: index.js [hunk 3]
```diff
@@ -111,19 +119,19 @@ function* run() {
...
-    } else {
-      // First time starting up, need to create a replica set config
-      for (const manager of rs.managers) {
-        yield manager.stop();
+      } else {
+        // First time starting up, need to create a replica set config
+        for (const manager of rs.managers) {
+          yield manager.stop();
```
Comment (vkarpov15) [inline index.js]: What's up with this extra indent? Not a big deal, just curious

Comment (chaiwa-berian) [reply index.js]: I am not sure I messed up the indents so I was trying to get them back to some good look! Thanks.

**Pull request merged**